scripts/core/usm.yml

```diff
@@ -98,19 +98,19 @@ etors:
       desc: "Hint that memory will be read from frequently and written to rarely"
     - name: CLEAR_READ_MOSTLY
       value: "$X_BIT(2)"
-      desc: "Removes the affect of ::$X_USM_ADVICE_SET_READ_MOSTLY"
+      desc: "Removes the affect of ::$X_USM_ADVICE_FLAG_SET_READ_MOSTLY"
```

Comment:
Looks like the generator scripts are automatcially inserting an extra `::` above, is it necessary to add them explicitly here?

Reply:
Done.